cdap-ui/app/cdap/components/Experiments/DetailedView/ModelStatusIndicator/index.js

```diff
@@ -30,21 +31,40 @@ const DEFAULT_STATUS_MAP = {
 };
 
 const STATUS_ICON_MAP = {
-  TRAINING: {
-    className: 'spin fa-spin',
-    icon: 'icon-spinner'
+  [MODEL_STATUS.SPLITTING]: {
+    className: 'fa-spin ',
+    icon: 'icon-spinner',
+    color: MODEL_STATUS_TO_COLOR_MAP[MODEL_STATUS.SPLITTING]
+  },
+  [MODEL_STATUS.SPLIT_FAILED]: {
+    className: '',
```

Comment:
Do we need these empty strings for `className`? It's ok if it's undefined right, if we don't need to pass the prop?

Reply:
I think its better to have `''` than undefined. `className` ignores `''` but renders a class class 'undefined' if its undefined.

Reply:
Huh interesting, didn't know that